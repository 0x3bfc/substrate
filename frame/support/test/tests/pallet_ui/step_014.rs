#[frame_support::pallet]
mod pallet {
	#[pallet::config]
	pub trait Trait: frame_system::Trait {}

	#[pallet::module]
	pub struct Module<T> {}

	#[pallet::module_interface]
	impl Foo {}
}

fn main() {
}
