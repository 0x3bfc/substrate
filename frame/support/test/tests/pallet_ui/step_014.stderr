error: Invalid module type: expect `Module<T>` or `Module<T, I>`
  --> $DIR/step_014.rs:10:7
   |
10 |     impl Foo {}
   |          ^^^

error: expected `Module`
  --> $DIR/step_014.rs:10:7
   |
10 |     impl Foo {}
   |          ^^^
